website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10.0
+description: Podman Desktop v0.10.0 has been released!
+slug: podman-desktop-release-0.10.0
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+Let's talk about all that is new with the latest release of Podman Desktop v0.10.0!
```

Comment:
- That's the first sentence of the blog post, we want to give people the desire to read the content and learn more about what's in. As a reader, this sentence does not give anything. 
- Why not trying to callout some of the most exciting features of the release again?